oop/associations/what/text.md

```diff
@@ -26,15 +26,15 @@ Objects in an OO solution need to be connected to each other to form a network s
 
 <box>
 
-{{ icon_example }} In our example, as some `Course` objects can have associations with some `Student` objects, we can view it as an association between the `Course` class and the `Student` class.
+{{ icon_example }} In our example, as some `Course` objects can have associations with some `Student` objects, youcan view it as an association between the `Course` class and the `Student` class.
```

Comment:
missing space

Reply:
@Tejas2805 Have you fixed this?

Reply:
Oh fixed it that day itself. But due to shifting from hostel last 2/3 days forgot to inform earlier. 

Can see the change [here](https://github.com/se-edu/se-book/pull/83/commits/e92d454b31bd9a0222975b5e24e1adb3bdee224b)

Reply:
Why is it not showing in the PR diff?